docs/reference/cluster/nodes-stats.asciidoc

```diff
@@ -536,97 +538,13 @@ that used an index shard as a target.
 (integer)
 Time in milliseconds
 recovery operations were delayed due to throttling.
-
-[[cluster-nodes-stats-api-response-body-fs]]
-===== `fs` section
```

Comment:
Note to reviewers:

This section was moved to line 851. No substantive changes were made.